docs/src/main/paradox/index.md

```diff
@@ -6,10 +6,10 @@ part of this project as well as links to components that are maintained by other
 
 The code in this documentation is compiled against
 
-* Alpakka $version$ ([Github](https://github.com/akka/alpakka))
+* Alpakka $version$ ([Github](https://github.com/akka/alpakka), [API docs](https://developer.lightbend.com/docs/api/alpakka/current/akka/stream/alpakka/index.html))
```

Comment:
linking to `current` instead of `$version$` might be debatable, but I like it! Looks like we only publish scaladoc, not javadoc? Added #629.

Reply:
I tried to use `$version$`, but Paradox does not resolve properties within URLs.